site/composer.json

```diff
@@ -13,7 +13,8 @@
     "phpunit/phpunit": "~4"
   },
   "require": {
-    "php": ">=5.5.0",
+    "php": "7.2.*",
```

Comment:
Why change PHP requirement?

Reply:
because 5.5  has not `posix` extension https://cloud.google.com/appengine/docs/standard/php/runtime

Reply:
I mean 7.2 is greater than 5.5, should work. Or GCP installs 5.5 because of this?